modules/tensor_mechanics/src/materials/ComputeMultiPlasticityStress.C

```diff
@@ -77,6 +77,9 @@ ComputeMultiPlasticityStress::ComputeMultiPlasticityStress(const InputParameters
     _my_elasticity_tensor(RankFourTensor()),
     _my_strain_increment(RankTwoTensor())
 {
+  if (_epp_tol <= 0)
+    mooseError("ComputeMultiPlasticityStress: ep_plastic_tolerance must be positive");
+
```

Comment:
Removing RangeCheckedParam in favour of an explicit check on _epp_tol in the constructor:

That tolerance is often set quite small (1E-12 is pretty standard), so the RangeChecked thing fails.  In that case i thought the explicit check was the simplest.